public/svgs/frost/round-add.svg

```diff
@@ -1,6 +1,10 @@
-<svg viewBox="0 0 20 20">
-	<rect x="9.5" y="4" fill="#009EEF" width="1" height="12"/>
-	<rect x="4" y="9.5" fill="#009EEF" width="12" height="1"/>
-	<path fill="#009EEF" d="M10,1c5,0,9,4,9,9s-4,9-9,9s-9-4-9-9S5,1,10,1z M10,17.9c4.3,0,7.9-3.5,7.9-7.9c0-4.3-3.5-7.9-7.9-7.9
-	S2.1,5.7,2.1,10C2.1,14.3,5.7,17.9,10,17.9z"/>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This gets into the composite svg concept - basically we want a round svg that we can use as a backing for any other svg.  If we don't already have an issue to look at this, please raise one.